src/app/watchlist/models/getwatchlistScripsModel.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+type Scrip struct {
+	Symbol string `json:"symbol"`
+	Token  uint   `json:"token"`
+}
+type GetWatchlistScrips struct {
+	WatchlistScrip []Scrip `json:"scrips"`
+}
```

Comment:
Add examples for all the fields

Reply:
It is response model of scrips after I am giving watchlist name in params.

Reply:
Still you have to add that, it will be added in the swagger documentation which will give us a sample success scenario example.